### ISSUE TITLE
VPN Support

### ISSUE THREAD

**Issue by Tcm9** [ShadowOxygen/OxygenX#20]
It should be possible to use NordVPN, HideMyAss or even your own VPN to check accounts. This will probably be slower than with proxies, but proxies are often banned or slow.  
With HideMyAss and NordVPN the ip should then simply be changed. With a private VPN the Vpn should be deactivated and then activated again each time.
private vpn in Windows 10:
![image](https://user-images.githubusercontent.com/31276706/86245961-9103c800-bbaa-11ea-97bb-692d5b5d8c66.png)

**Comment by Tcm9** [comment]
Another possibility is the "Router reconnect" script for the Fritzbox.

**Comment by ghost** [comment]
You can use vpn. Just put the use of proxy's as "false"

**Comment by Tcm9** [comment]
yes but the ip does not change automatically after a few accounts are checked.

**Comment by ShadowOxygen** [comment]
Uh so you can use VPN, but I do not know how to switch vpn using python

But a good idea

**Comment by Tcm9** [comment]
Private vpn: (cmd)
rasdial [vpn name] /DISCONNECT

With the fritzbox script the exe only has to be executed each time.

Hide my ass vpn: https://support.hidemyass.com/hc/en-us/articles/202723586-CLI-Controlling-HMA-VPN-client-externally-with-parameters

NordVPN: https://support.nordvpn.com/Connectivity/Windows/1350897482/Connect-to-NordVPN-app-on-Windows-using-the-Command-Prompt.htm

I hope I could help.
